lms/djangoapps/completion_api/tests/test_views.py

```diff
@@ -204,3 +204,80 @@ def test_staff_access(self):
         self.client.force_authenticate(user)
         response = self.client.get('/api/completion/v0/course/?user={}'.format(self.test_user.username))
         self.assertEqual(response.status_code, 200)
+
+
+@override_settings(STUDENT_GRADEBOOK=True)
+class CompletionBlockUpdateView(SharedModuleStoreTestCase):
```

Comment:
This class name should end in `TestCase` - otherwise it sounds like the actual view.